src/browser-locale/index.js

```diff
@@ -0,0 +1,64 @@
+/* global TERRA_AGGREGATED_LOCALES */
+
+// The fallback locale that will be used if a supported locale is unable to be retrieved from the browser.
+const DEFAULT_LOCALE = 'en';
+
+/**
+ * Returns a boolean indicating if the locale is supported.
+ * @param {string} locale - The locale.
+ * @returns {bool} - A boolean indicating the locale is supported.
+ */
+const isSupported = (locale) => {
```

Comment:
Should we be ensuring the string format is correct? On your spike, it links to: https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language

it says safari <10.2 returns country code as lowercase
> Note that in Safari on iOS prior to 10.2, the country code returned is lowercase: "en-us", "fr-fr" etc.

I don't think we support that version version but just curious if we need to consider it